pkg/util/pem_test.go

```diff
@@ -116,6 +119,44 @@ func TestValidateAndSanitizePEMBundle(t *testing.T) {
 			}
 		})
 	}
+
+	expiredCertCases := map[string]struct {
+		parts          []string
+		expectedOutput []string
+		expectErr      bool
+	}{
+		"valid bundle with valid certs and an expired cert": {
+			parts:          []string{dummy.TestCertificate1, dummy.TestExpiredCertificate, dummy.TestCertificate3},
+			expectedOutput: []string{dummy.TestCertificate1, dummy.TestCertificate3},
+			expectErr:      false,
+		},
```

Comment:
comment: I think it'd be good to have a test that the expired cert is _not_ removed if FilterExpired is false, but I don't think it should block this PR: I'll raise a followup PR to add that after this merges :)

Reply:
Oh wait, the cases above test that - you already have it tested 😁 Thank you!